atom/browser/api/atom_api_app.cc

```diff
@@ -844,6 +848,8 @@ void App::BuildPrototype(
       .SetMethod("setUserTasks", base::Bind(&Browser::SetUserTasks, browser))
       .SetMethod("getJumpListSettings", &App::GetJumpListSettings)
       .SetMethod("setJumpList", &App::SetJumpList)
+      .SetMethod("getSystemAccentColor",
```

Comment:
Moving the method to `systemPreferences` is relatively easy.  The issue is that the event `WM_DWMCOLORIZATIONCOLORCHANGED` is emitted at the window level, and I don't think we should be coupling the `NativeWindow` class with `systemPreferences`.